_sources/SequenceMutation/NonmutatingMethodsonStrings.rst

```diff
@@ -0,0 +1,111 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: seqmut-8-
+   :start: 1
+
+Non-mutating Methods on Strings
+===============================
+
+The "dot notation" is the way we connect an object to one of its attributes or to invoke a method on 
```

Comment:
This seems a little odd to review the dot notation at this point.  I think we could just delete this sentence.